proxstar/static/js/script.js

```diff
@@ -880,6 +875,81 @@ $("#edit-boot-order").click(function(){
     });
 });
 
+function renderBootOrder(boot_order) {
+    let options = document.createElement('div');
+    if (boot_order.legacy) {
+        for (i = 0; i < boot_order.order.length; i++) {
+            text = document.createElement('span');
+            text.innerHTML = `${i + 1}. `;
+            options.append(text);
+            var entry = document.createElement('select');
+            entry.setAttribute("id", `boot-order-${i + 1}`);
+            for (j = 0; j < boot_order.order.length; j++) {
+                entry.appendChild(new Option(boot_order.order[j].device, boot_order.order[j].device));
+            }
+            entry.selectedIndex = i;
+            entry.setAttribute('style', 'width: 85px');
+            options.append(entry);
+            options.append(document.createElement('br'));
+        }
+    }
+    else {
+        let table = document.createElement('table');
+        table.classList.add('table', 'table-sm', 'borderless', 'text-left');
+        let thead = table.createTHead();
+        thead.classList.add('font-weight-bold');
+        let tbody = table.createTBody();
+        tbody.classList.add('text-break', 'boot-order-sortable');
+        tbody.id = 'boot-order-sortable';
+        let hrow = thead.insertRow();
+        hrow.insertCell().innerHTML = 'Order';
+        hrow.insertCell().innerHTML = 'Enabled';
+        hrow.insertCell().innerHTML = 'Device';
+        hrow.insertCell().innerHTML = 'Description';
+        for (i = 0; i < boot_order.order.length; i++) {
+            let row = tbody.insertRow();
+            row.id = `boot-order-${i + 1}`;
+            $(row.insertCell()).append(
+                $('<i>', {
+                    class: 'fas fa-bars'
+                }),
+                $('<span>', {
+                    class: 'boot-order-number',
+                    id: `boot-order-number-${i + 1}`,
+                    style: 'margin-left: .25rem',
+                    text: `${i + 1}`
+                })
+            );
+            let checkCell = $(row.insertCell()).addClass('text-center');
+            $(checkCell).append(
+                $('<input>', {
+                    type: 'checkbox',
+                    class: 'form-check-input boot-order-check',
+                    checked: boot_order.order[i].enabled
+                })
+            );
+            row.insertCell().innerHTML = boot_order.order[i].device;
+            row.insertCell().innerHTML = boot_order.order[i].description;
```

Comment:
According to @harmonherring the boot order selection menu descriptions break the view on mobile devices.